README.md

```diff
@@ -345,3 +345,14 @@ If you want to contribute to this project, see [Contributing Guide](https://gith
 | Boilerplate            | a lot: <ul><li>ADTs requires amount of annotation equal to or exceeding the actual type definitions</li><li>requires explicit serializers and deserializers in certain cases (e.g. enums)</li></ul>                                                                                                                                                                                             | every top-level sealed trait must be registered manually - but see [Codec Registration Checker](#missing-codec-registration)   | in case of custom types, a second layer of models is needed                                    | sometimes requires annotations                                                                   | every top-level sealed trait must be registered manually; every transitively included class must have an explicitly defined codec | every top-level sealed trait must be registered manually                                                                                                     |
 | Schema evolution       | <ul><li>removing field</li><li>adding optional field</li></ul> with [`JacksonMigration`](https://doc.akka.io/docs/akka/current/serialization-jackson.html#schema-evolution): <ul><li>adding mandatory field</li><li>renaming field</li><li>renaming class</li><li>support of forward versioning for rolling updates</li></ul>| <ul><li>adding optional field</li><li>removing optional field</li><li>adding required field with default value</li><li>removing required field</li><li>renaming field</li><li>reordering fields</li><li>transforming data before deserialization</li></ul> | <ul><li>adding optional field</li><li>removing optional field</li><li>adding required field with default value</li><li>removing required field</li><li>renaming field</li><li>reordering fields</li><li>changing between compatible types</li></ul>  | <ul><li>reordering fields</li><li>renaming fields</li><li>adding optional field</li><li>adding required field with default value</li><li>removing field with default value</li></ul> | <ul><li>renaming fields</li><li>transforming data before deserialization</li></ul> | <ul><li>adding field</li><li>removing field</li><li>renaming field</li><li>renaming class</li></ul> |
+
+## Appendix B: what happens with serialization of Messages / Events / States that do not extend the base trait bound to the serializer
+
+In runtime, if given Message/Event/State class does not extend the base trait bound to serializer, following problems will occur (depending on Akka version and settings):
```

Comment:
```suggestion
In runtime, if given Message/Event/State class does not extend a base trait bound to serializer, the following problems will occur (depending on Akka version and settings):
```
(_nie rozszerza jakiegoś/pewnego traitu_ &mdash; doesn't need to be **the** specific one)

Reply:
done